README.md

```diff
@@ -216,19 +216,13 @@ minikube start --kubernetes-version=<version of your Kubernetes clusters, exampl
 
 ### Run tests
 
-To run all tests:
-
-```
-make tests
```

Comment:
can no longer offer this. it runs unit and integrations tests in tandem but the 2 suites conflict on HOME env. so each suite must be run separately.